team-config.yml

```
@@ -94,3 +94,12 @@ help-with-fees:
     build_notices_channel: "#pet-devops"
   tags:
     application: help-with-fees
+Child Arrangement Service:
```

this should be the 'product' not display name
```suggestion
c100:
```

---

Thanks, Tim. Changed it to product name.